src/utils/request.ts

```diff
@@ -374,9 +374,11 @@ export function getRequestIP(
   if (opts.xForwardedFor) {
     // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For#syntax
     const _header = event.req.headers.get("x-forwarded-for");
-    const xForwardedFor = (_header || "")?.split(",").shift()?.trim();
-    if (xForwardedFor) {
-      return xForwardedFor;
+    if (_header) {
+      const xForwardedFor = _header.split(",")[0].trim();
+      if (xForwardedFor) {
+        return xForwardedFor;
+      }
     }
```

Comment:
If we want to code-golf this we can use `&&`, do you prefer that I write it like this?

```suggestion
    const xForwardedFor = _header && _header.split(",")[0].trim();
    if (xForwardedFor) {
      return xForwardedFor;
    }
```

(this may have a slight, but not noticeable impact, as in case `_header` is falsy it will have to be evaluated as falsy twice)

Reply:
explicit check version seems better